Server/src/controllers/authenticationController.ts

```diff
@@ -70,8 +72,14 @@ export class AuthenticationController {
 
   async resetPasswordWithResetTokenRequest(request: Request, response: Response, next: NextFunction): Promise<Response> {
     this.invalidResponse = this.validateResetPasswordRequest(request);
-    if (this.invalidResponse) {
+    if (this.invalidResponse) {//to do: change to proper message
       return response.status(400).json("User was not found");
+    }//checks of the schema is valid
+    if (passwordSchema.isValidSync(request.body.password)) {
+      return
+    }//checks if the passwords match
+    if (this.passwordConfirmationRequest(request)) {
+      return
```

Comment:
What happens in this return

Reply:
fixed on the latest commit with proper return status